config/locales/administrate.zh-TW.yml

```diff
@@ -0,0 +1,23 @@
+---
+zh-TW:
+  administrate:
+    actions:
+      confirm: 你確定？
+      destroy: 刪除
+      edit: 編輯
+      show: 檢視
+    controller:
+      create:
+        success: "已成功新增 %{resource}。"
+      destroy:
+        success: "已成功刪除 %{resource}。"
+      update:
+        success: "已成功更新 %{resource}。"
+    fields:
+      has_many:
+        more: 顯示 %{total_count} 筆中的 %{count} 筆資料
```

Comment:
I haven't tried administrate yet, could you point me where this translation being used from the demo site or a screenshot? :blush: